src/mlpack/methods/ann/gan/gan.hpp

```diff
@@ -95,16 +96,25 @@ class GAN
   //! Move constructor.
   GAN(GAN&&);
 
+  /**
+   * Prepare the network for the given data.
```

Comment:
Maybe it would be useful to be more precise here, we could add initialise the weights, generator, discriminator?

Reply:
Okay, I have changed it to `Intialize the generator, discriminator and weights of the model for training.`. Let me know about it.